backend/tests/regression/answer_quality/cli_utils.py

```diff
@@ -0,0 +1,291 @@
+    else:
+        print("Unsupported operating system. Please start Docker manually.")
+        sys.exit(1)
+
+    print("Waiting for Docker to start...")
+    while not check_docker_running():
+        time.sleep(1)
+    print("Docker has been started successfully.")
+
+
+def switch_to_branch(branch: str) -> None:
+    print(f"Switching to branch: {branch}...")
+    returncode, _, stderr = _run_command(f"git checkout {branch}")
+
+    if returncode == 0:
+        # update_repository()
+        print(f"Successfully switched to branch: {branch}")
+    else:
+        print(f"Failed to switch to branch: {branch}")
+        print(f"Error: {stderr}")
+        sys.exit(1)
+
+
+def update_repository() -> None:
+    print("Updating the repository...")
+    returncode, _, stderr = _run_command("git pull")
+
+    if returncode == 0:
+        print("Repository updated successfully.")
+    else:
+        print(
+            "Failed to update the repository. Please check your internet connection and try again."
+        )
+        print(f"Error: {stderr}")
+        sys.exit(1)
+
+
+def set_volume_location_env_variables(
+    suffix: str, base_path: str, use_cloud_gpu: bool
+) -> str:
+    # Use the user's home directory as the base path
+    target_path = os.path.join(os.path.expanduser(base_path), f"test{suffix}")
+    directories = {
+        "DANSWER_POSTGRES_DATA_DIR": os.path.join(target_path, "postgres/"),
+        "DANSWER_VESPA_DATA_DIR": os.path.join(target_path, "vespa/"),
+    }
+    if not use_cloud_gpu:
+        directories["DANSWER_INDEX_MODEL_CACHE_DIR"] = os.path.join(
+            target_path, "index_model_cache/"
+        )
+        directories["DANSWER_INFERENCE_MODEL_CACHE_DIR"] = os.path.join(
+            target_path, "inference_model_cache/"
+        )
+
+    # Create directories if they don't exist
+    for env_var, directory in directories.items():
+        os.makedirs(directory, exist_ok=True)
+        os.environ[env_var] = directory
+        print(f"Set {env_var} to: {directory}")
+    relari_output_path = os.path.join(target_path, "relari_output/")
+    os.makedirs(relari_output_path, exist_ok=True)
+    return relari_output_path
+
+
+def set_env_variables(
+    remote_server_ip: str,
+    remote_server_port: str,
+    use_cloud_gpu: bool,
+    openai_api_key: str,
+) -> None:
+    seed_json = {
+        "llms": [
+            {
+                "name": "Netflix NCP",
+                "provider": "openai",
+                "api_key": openai_api_key,
+                "default_model_name": "gpt-4o",
+                "model_names": ["gpt-4o"],
+            }
+        ],
+    }
+    env_vars: dict = {}
+    env_vars["ENV_SEED_CONFIGURATION"] = json.dumps(seed_json)
+    env_vars["ENABLE_PAID_ENTERPRISE_EDITION_FEATURES"] = "true"
+    if use_cloud_gpu:
+        env_vars["MODEL_SERVER_HOST"] = remote_server_ip
+        env_vars["MODEL_SERVER_PORT"] = remote_server_port
+
+    for env_var, directory in env_vars.items():
```

Comment:
why is this called directory? Also no point in gathering it all in a dict, just assign each to os.environ as you hit it